code/modules/mob/mob_defines.dm

```diff
@@ -46,6 +46,7 @@
 	var/cultslurring = 0	//Carbon
 	var/real_name = null
 	var/druggy = 0			//Carbon
+	var/disgust = 0 		//Carbon
```

Comment:
Sorry, I followed the pattern because I'm a bit shit at this still. :^) I'll move it up and possibly also move druggy up there once I move that to an organ.